modules/silauth/src/Auth/Source/auth/AuthError.php

```diff
@@ -17,7 +17,7 @@ class AuthError
     const CODE_RATE_LIMIT_MINUTES = 'rate_limit_minutes';
 
     private string $code;
-    private array $messageParams = [];
+    private array $messageParams;
```

Comment:
If you don't initialize and then you access before it is set, PHP throws an error.  Is there a nullable case?

Reply:
Hmm, didn't think about that. PHP is tricky. Thanks for the heads-up.